android/assets/jsons/Translations/Other.json

```diff
@@ -1825,18 +1833,15 @@
 		French:"Ville la plus proche"
 		Romanian:"Cel mai apropiat oraș"
 		Spanish:"Ciudad más cercana"
-		Simplified_Chinese:"最靠近的城市"
+		Simplified_Chinese:"最近的城市"
 		Portuguese:"Cidade mais próxima"
 		German:"Nächstgelegene Stadt"
 		Czech:"Nejbližší město"
 }
 
 	"Action":{ // Overview -> Units - Action (Header)
 		Czech:"Činnost"
-}
-
-	"automation":{ // Overview -> Units - Action (column)
```

Comment:
it's not used when i change "automation" with "Automate" in "core/src/com/unciv/Constants.kt".